src/domain/flow-system/domain/src/entities/flow_configuration/flow_configuration_rule.rs

```diff
@@ -18,6 +18,7 @@ pub enum FlowConfigurationRule {
     Schedule(Schedule),
     BatchingRule(BatchingRule),
     CompactionRule(CompactionRule),
+    IngestRule(IngestRule),
```

Comment:
Why do we need previous `Schedule` branch ?

Reply:
It is still using for system flow